home/models.py

```diff
@@ -0,0 +1,60 @@
+from django.db import models
+
+from accounts.models import CustomUser as User
+from showcase.models import SingletonModel
+
+
+class Review(models.Model):
+    """ Participant Review to display on the home page """
+    created = models.DateTimeField(auto_now_add=True)
+    updated = models.DateTimeField(auto_now=True)
+
+    created_by = models.ForeignKey(User, null=True, blank=True,
+                                   on_delete=models.CASCADE,
+                                   related_name="reviews")
+    content = models.TextField()
+    rating = models.FloatField(null=True, blank=True)
+    visible = models.BooleanField(default=True)
+
+    def __str__(self):
+        return f'{self.created} - {self.content}'
+
+    class Meta:
+        verbose_name = 'Review'
+        verbose_name_plural = 'Reviews'
+
+
+class PartnershipRequest(models.Model):
+    """ A record from the Parntership Request form on the home page """
+    created = models.DateTimeField(auto_now_add=True)
+    updated = models.DateTimeField(auto_now=True)
+
+    company = models.CharField(max_length=255)
+    email = models.CharField(max_length=255)
+    phone = models.CharField(max_length=255)
+    contact_name = models.CharField(max_length=255)
+    description = models.TextField()
+
+    def __str__(self):
+        return f'{self.created} - {self.company}'
+
+    class Meta:
+        verbose_name = 'PartnershipRequest'
+        verbose_name_plural = 'PartnershipRequests'
+
+
+class PartnershipRequestEmailSiteSettings(SingletonModel):
+    """ Model to configure emails for the PartnershipRequest form """
+    from_email = models.CharField(max_length=255)
+    to_emails = models.TextField(
+        default='[]',
+        help_text=('This should be a list of emails (e.g. ["email1", ...]. '
```

Comment:
Possible future feature: improve this so that it's less error-proof. Currently it looks like it's expecting emails to be `comma-separated`, but what if they use `semicolons` or just "spaces" for example.

Reply:
It is intended to take that specific format so that it can be loaded into json straight away. I also added the example to the `help_text` to clarify it and added error handling in the function which sends the email if it cannot parse the string to json.